### ISSUE TITLE
osgEarth failed to build on 10.9.5

### ISSUE THREAD

**Issue by edgardomortiz** [Homebrew/homebrew-science#1874]
Here is the gist: https://gist.github.com/anonymous/ad1c35901fd359501610

I was trying to "install qgis-28 --with-globe --with-grass7 --with-orfeo --with-postgis --with-qt-mysql --with-r --with-saga-gis", and stops at the compilation of osgEarth

Any help on how to solve the issue is greatly appreciated

**Comment by iml** [comment]
I can reproduce the error, and it seems to occur with 2.6 as well. Seems to have been fixed in `HEAD`, though.

**Comment by edgardomortiz** [comment]
Hello, did you mean to try this? `<brew install --HEAD osgearth>`

It returns the following error:

==> Installing osgearth from homebrew/homebrew-science
==> Cloning https://github.com/gwaldron/osgearth.git
Updating /Library/Caches/Homebrew/osgearth--git
==> Checking out branch master
==> Patching
patching file CMakeModules/FindV8.cmake
Hunk #1 FAILED at 21.
Hunk #2 FAILED at 40.
2 out of 2 hunks FAILED -- saving rejects to file CMakeModules/FindV8.cmake.rej
patching file CMakeModules/FindJavaScriptCore.cmake
Hunk #1 FAILED at 21.
1 out of 1 hunk FAILED -- saving rejects to file CMakeModules/FindJavaScriptCore.cmake.rej
patching file CMakeModules/FindLibNoise.cmake
Hunk #1 FAILED at 43.
1 out of 1 hunk FAILED -- saving rejects to file CMakeModules/FindLibNoise.cmake.rej
Error: Failure while executing: /usr/bin/patch -g 0 -f -p1

**Comment by iml** [comment]
Yeah the patches need to be removed, though. I opened a PR in #1881 that bumps the version and removes the upstreamed patches but there's an outstanding issue with OSG compatibility. You could try HEAD with that, or edit the existing formula and comment out `patch :DATA`. For the former approach:
```
brew install --HEAD https://raw.githubusercontent.com/iml/homebrew-science/osgearth-26/osgearth.rb

**Comment by edgardomortiz** [comment]
Thank you, osgearth finishes compiling with this method but qgis-28 doesn't finish its compilation when the `<--with-globe>` option is enabled

**Comment by dpo** [comment]
Should be fixed in #2713.
